docs/Basics-Layout.md

```diff
@@ -63,7 +63,7 @@ class AwesomeProject {
   }
 };
 
-AppRegistry.registerComponent('AwesomeProject', () => AwesomeProject);
+AppRegistry.registerComponent('AwesomeProject', () => JustifyContentBasics);
```

Comment:
can you make the registerComponent stuff use the same name as the name of the class?